index.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.14.0/css/all.css"
+      integrity="sha384-HzLeBuhoNPvSl5KYnjx0BT+WB0QEEqLprO+NBkkk5gbc67FTaL7XIGa2w1L0Xbgc"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="./css/normalizer.css" />
+    <link rel="stylesheet" href="./css/styles.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+    <title>Document</title>
+  </head>
+  <body>
+    <div class="body">
+      <header class="header-section">
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav-item">
+              <a  class="white-color"  href="#"><i class="fab fa-apple"></i></a>
+            </li>
+            <li class="nav-item"><a class="white-color"  href="#">Store</a></li>
+            <li class="nav-item"><a  class="white-color" href="#">Mac</a></li>
+            <li class="nav-item"><a  class="white-color" href="#">Ipod</a></li>
+            <li class="nav-item"><a  class="white-color" href="#">Iphone</a></li>
+            <li class="nav-item"><a  class="white-color" href="#">Ipad</a></li>
+            <li class="nav-item"><a  class="white-color" href="#">Itunes</a></li>
+            <li class="nav-item"><a  class="white-color" href="#">support</a></li>
```

Comment:
- [ ] Instead of making only the text clickable, make the whole rectangle clickable ([link to page](https://web.archive.org/web/20140301004610/http://www.apple.com/))